src/components/CryptoCards/CryptoCards.tsx

```diff
@@ -0,0 +1,84 @@
+import { Icon } from '../Icon';
+import { CryptoCardProps } from './types';
+import { iconTypes } from '../Icon/collection';
+import { cryptoCardStyles } from './CryptoCards.styles';
+import React from 'react';
+import styled from 'styled-components';
+import color from '../../styles/colors';
+
+const CryptoCardStyled = styled.div`
+    ${cryptoCardStyles.borderStyle}
+`;
+
+const ButtonStyles = styled.button`
+    ${cryptoCardStyles.buttonStyle}
+`;
+
+const BtnTxtStyle = styled.p`
+    ${cryptoCardStyles.btnTxtStyle}
+`;
+
+const InfoStyle = styled.div`
+    ${cryptoCardStyles.infoStyle}
+`;
+
+const NetworkInfoHolder = styled.div`
+    ${cryptoCardStyles.networkInfoHolder}
+`;
+
+const NetworkStyle = styled.p`
+    ${cryptoCardStyles.networkStyle}
+`;
+
+const ChainNameStyle = styled.p`
+    ${cryptoCardStyles.chainNameStyle}
+`;
+
+const ImageBoxStyle = styled.div`
+    ${cryptoCardStyles.imageBoxStyle}
+`;
+
+const Logo = styled.div`
+    ${cryptoCardStyles.logo}
+`;
```

Comment:
Please move all styled consts to styles.ts and try to keep component file clean from css/styles